content/en/docs/started/workstation/kubeflow-on-microk8s.md

```diff
@@ -20,26 +22,25 @@ To get Kubeflow running using MicroK8s, you'll need to install MicroK8s, enable
 
 **Note:** You need MicroK8s version 1.18 and above to enable and run Kubeflow.
 
-1. Install MicroK8s with [Snap](https://snapcraft.io/) by running the following command:
+1. Install MicroK8s by running the following command:
 
     ```
     sudo snap install microk8s --classic
     ```
 
-2. Verify that MicroK8s is running:
+2. Add current user to admin group:
```

Comment:
@RFMVasconcelos Hi! Since you're using `$USER`, I assume the "current user" is the reader:

```suggestion
2. Add yourself (the current user) to admin group:
```

Reply:
Makes sense :)

Reply:
Fixed!